src/base/utils/misc.cpp

```diff
@@ -243,6 +243,14 @@ QString Utils::Misc::unitString(Utils::Misc::SizeUnit unit)
                                        units[static_cast<int>(unit)].source, units[static_cast<int>(unit)].comment);
 }
 
+QString Utils::Misc::paddedUnitString(Utils::Misc::SizeUnit unit, bool isSpeed)
```

Comment:
The main goal was to reuse already translated strings. Also I thought that unit strings are always prepended by space to separate from numbers. So at first I changed existing `unitString()` to use `isSpeed` parameter (as you want) and do padding (to follow DRY principle and so on). But build broke up. There is 8 usage point in src/gui/search/searchjobwidget.cpp on lines 319-325 which are ignored by qtcreator code parser "find usages" function (because of lack of Utils::Misc:: access specification) but are  compilable nevertheless. (I think they should generate compile error, but they didn't).
This forced me to resurrect old  `unitString()` and place a new function side by side. And this function is used in 4 places, so it is not so useless in spite of it's simplicity.

Reply:
It doesn't generate compile errors due to [ADL](https://en.cppreference.com/w/cpp/language/adl). Personally, I don't like the "blind" use of such things.

Reply:
>And this function is used in 4 places, so it is not so useless in spite of it's simplicity.

That sounds lame...
If it will be used some times in parentheses, will you add "unitStringInParentheses()" and so on?
>So at first I changed existing unitString() to use isSpeed parameter (as you want) and do padding (to follow DRY principle and so on). But build broke up. There is 8 usage point in src/gui/search/searchjobwidget.cpp on lines 319-325 which are ignored by qtcreator code parser "find usages" function 

If you add default (false) value to  isSpeed param there shouldn't be problems with existing usages.

Reply:
> will you add "unitStringInParentheses()" 

At least I will consider to do this. :)
Removed paddedUnitString.